.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Create Relase & Publish Package
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: false
+          prerelease: false
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
```

Comment:
Is version 12 still up to date?

Reply:
Didn't we upgrade to version 16

Reply:
Yes, I copied this from the internationalization package. I will update there as well.